src/ui/control/attribution_control.js

```diff
@@ -66,11 +66,12 @@ class AttributionControl {
     }
 
     _updateEditLink() {
-        if (!this._editLink) this._editLink = this._container.querySelector('.mapboxgl-improve-map');
+        if (!this._editLink) this._editLink = this._container.querySelector('.mapbox-improve-map');
```

Comment:
So, if I'm understanding this right, the change from `.mapboxgl-improve-map` to `.mapbox-improve-map` is in order to match against the CSS class used in our the TileJSON provided by Mapbox's vector tile sources?